scripts/verdaccio.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
```

Comment:
Not sure, but you might want `set -e` here? That way if any command fails, we don't continue with the following ones? Or does the `trap` below handle that?

Reply:
Good tip, I'll set that just in case the errexit behavior bails out if a command fails (it seems to have already been behaving like this in my experience but looking it up seems to indicate that it is false by default)
The `trap` below handles running the cleanup function upon receiving the `ERR` or `EXIT` signal.